pkg/management/postgres/webserver/webserver.go

```diff
@@ -32,6 +32,10 @@ const (
 	DefaultReadTimeout = 20 * time.Second
 	// DefaultReadHeaderTimeout is the default value to be used by the webservers
 	DefaultReadHeaderTimeout = 3 * time.Second
+	// DefaultWriteTimeout is the default value to be used by the webservers
+	DefaultWriteTimeout = 60 * time.Second
```

Comment:
Need to check with team if we have any endpoints that aren't expected to return within 60 seconds during normal operation. @sxd ?